requirements/constraints.txt

```diff
@@ -16,3 +16,7 @@ requests==2.22.0
 
 # quality (jsonschema, pytest - 2.0.0) vs. travis (tox, virtualenv - 1.7.0)
 importlib-metadata==1.7.0
+
+pyopenssl==19.1.0
+ 
+diff-cover<4.2.0
```

Comment:
I had to add these as requirements constraints in order to have successful run of dbt. Should I push them into repository as well ?

Reply:
Yes, we'll need that. Can you comment there which upstream things caused the pinning? That way we'll know when to remove them later.

Reply:
Sure, added the description @bmedx